docs/javascript/v2/features/hls.mdx

```diff
@@ -181,6 +181,77 @@ function HLSView({ hlsUrl }) {
 
 > **Note:** If you wish to customize the stream, please check [this guide](./../foundation/live-streaming#step-4-customize-the-stream) for more information.
 
+## Changing the quality of Stream(Hls.js)
+
+You may want to change the quality layer of the stream based on user input or some other criteria. Each library handles this differently.
+Below we mention an example snippet on how to do it if you choose to use hls.js. Please refer to the respective documentations of the library
+you use.
+
+<Tabs id="hlsChangeQuality" items={['JavaScript', 'React']} />{' '}
+
+<Tab id='hlsChangeQuality-0'>
+
+```js
+import Hls from 'hls.js';
+
+let hls = null;
+const video = document.getElementById('video');
+const browserHasNativeHLSSupport = videoRef.current.canPlayType('application/vnd.apple.mpegurl');
```

Comment:
`video.canPlayType.` btw, this doesn't seem to be used anywhere?

Reply:
done